documentation/design-docs/dotnet-tools.md

```diff
@@ -447,52 +436,49 @@ COLLECT
 
 
      Examples:
-       > dotnet-dump collect --process-id 1902 --number 2 --type mini --output ~/dumps/go/here/
-       Writing:     ~/dumps/go/here/core0001
-    
-... 10 seconds pass (the default time interval)
-
-       > dotnet-dump collect --process-id 1902 --number 2 --type mini --output ~/dumps/go/here/
-       Writing:     ~/dumps/go/here/core0001
-       Writing:     ~/dumps/go/here/core0002
-       Complete
 
+        $ dotnet dump collect --process-id 1902 --number 2 --type mini
+        Writing minidump to file ./core_20190226_135837
+        Written 98983936 bytes (24166 pages) to core file
+        Complete
+        
+        $ dotnet dump collect --process-id 1902 --number 2 --type mini
+        Writing minidump to file ./core_20190226_135850
+        Written 98959360 bytes (24160 pages) to core file
+        Complete
 
 ANALYZE
 
     dotnet-dump analyze [-h|--help] dump_path
 
     Starts an interactive shell with debugging commands to explore a dump
 
-    -h, --help
```

Comment:
I don't think we should remove the help option

Reply:
See above.